demobot.py

```diff
@@ -1,110 +1,73 @@
 import os
 
 import logging
-import shelve
-
-from discord.ext.commands import Context
-from dotenv import load_dotenv
 
 from discord.ext import commands
+from discord.ext.commands import Context, DefaultHelpCommand
+from dotenv import load_dotenv
 
+# set logging level todo get this from .env file
 logging.basicConfig(level=logging.INFO)
 
+# load the private discord token from .env file.
 load_dotenv()
 TOKEN = os.getenv('DISCORD_TOKEN')
 
-bot = commands.Bot(command_prefix='!')
 
-queues = {'dummy':[]}
+class General(commands.Cog):
+    """
+    General commands for everyone.
+    """
 
-adminRoles = ['Demonstrator','demonstrator','Admin role','ADMIN ROLE','DEMONSTRATOR','admin role','adminrole']
+    @commands.command()
+    async def source(self, ctx: Context):
+        """
+        Link to the sourcecode.
+        """
+        await ctx.send('https://github.com/IdrisTheDragon/demoHelperBot')
 
 
-def getQueue(serverName:str):
-    if serverName in queues.keys():
-        return queues.get(serverName)
-    else:
-        queues[serverName] = []
-        return queues.get(serverName)
+# Initialise the Bot object with an accesible help Command object
+helpCommand = DefaultHelpCommand()
+bot = commands.Bot(
+    command_prefix='!',
+    help_command=helpCommand,
+    description='DemoHelper is a queue system for online practicals.\n Students can add themselves to the queue.\n '
+                'When a demonstrator is free to help, they can call the next command to get the next waiting student. '
```

Comment:
I like this idea but I think it would be better to have this under a separate command called !info instead of convoluting the help command as this could be called often by students and demonstrators